### PR TITLE
Fix issue #59 - Update contributing.md doc

### PR DESCRIPTION
**Pull request by Trifall**
This PR addresses the issue outlined in `Improve 'Get Started Deving' Onboarding` #59 

I have updated the CONTRIBUTING.md file to reflect the transition from 'vscode' to 'pearai-app'.  Most links have been modified to point to the pearai-app repository, unless it is specific to VSCodes troubleshooting or build fixes, to which I have left them unchanged. 

There are still some things that need to be rewritten or removed.

**Sections** in the doc that need modifying or removal (that I can see):

- Introducing usage of new Electron API with a PR: Replace `VS Code` with `PearAI App`?
- Spell check errors: Language features; may need to be written, link is for original wiki.
- Enable Commit Signing: removal?
- Debugging: Rewrite working. Should `VS Code `be replaced with `PearAI App`?
- Where to Contribute: performance / user experience / architectural needs a rewrite

Additionally:
- Links: All wiki links point to VSCode's original repo still, as the pearai-app repo doesn't have those pages. #

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -249,42 +254,51 @@ The **extension host** process runs code implemented by a plugin. To debug exten
...
 
 To enable us to quickly review and accept your pull requests, always create one pull request per issue and [link the issue in the pull request](https://github.com/blog/957-introducing-issue-mentions). Never merge multiple requests in one unless they have the same root cause. Be sure to follow our [[Coding Guidelines|Coding-Guidelines]] and keep code changes as small as possible. Avoid pure formatting changes to code that has not been modified otherwise. Pull requests should contain tests whenever possible.
 
 ### Introducing usage of new Electron API with a PR
+
 A pull request that depends on Electron API that VS Code is currently not using comes with a certain risk and may be rejected. Whenever we update Electron, there is a chance that less popular Electron APIs break and it is very hard to find out upfront. Once a PR lands in VS Code, the role of maintaining the feature moves to the team and as such we have to follow up with upstream components to ensure the feature is still supported. As such, as a rule of thumb:
```
Comment (Trifall) [inline CONTRIBUTING.md:288]: This section probably needs to be reworded.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,11 +32,11 @@ PearAI is a fork of VSCode, so simply follow VSCode's guide for running the app.
 
 The extension can be run in two ways:
 
-RECOMMENDED: Internally within the main PearAI application (which is a VSCode fork): https://github.com/trypear/pearai/. This guide is for running it internally.
+RECOMMENDED: Internally within the main PearAI application (which is a VSCode fork): <https://github.com/trypear/pearai/>. This guide is for running it internally.
 
 Standalone as an extension. For running it standalone, you will want to `cd` into `extensions/pearai-extension` and visit [Contributing to pearai-extension](extensions/pearai-extension/CONTRIBUTING.md)
```
Comment (Fryingpannn) [inline CONTRIBUTING.md:37]:
```suggestion
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -111,42 +113,42 @@ Otherwise, if you're a member of the VS Code team, follow the [Commit Signing](h
...
 ```
-cd vscode
+cd pearai-app
 yarn
 ```
 
 Then you have two options:
 
-- If you want to build from inside VS Code, you can open the `vscode` folder and start the build task with <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> (<kbd>CMD</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> on macOS). The build task will stay running in the background even if you close VS Code. If you happen to close VS Code and open it again, just resume the build by pressing <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> (<kbd>CMD</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd>) again. You can kill it by running the `Kill Build VS Code` task or pressing <kbd>Ctrl</kbd>+<kbd>D</kbd> in the task terminal.
+- If you want to build from inside VS Code, you can open the `pearai-app` folder and start the build task with <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> (<kbd>CMD</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> on macOS). The build task will stay running in the background even if you close the app.
```
Comment (Fryingpannn) [inline CONTRIBUTING.md:151]:
```suggestion
- If you want to build from inside VS Code, you can open the `pearai-app` folder and start the build task with <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> (<kbd>CMD</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> on macOS). The build task will stay running in the background even if you close the app. If you happen to close VS Code and open it again, just resume the build by pressing <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd> (<kbd>CMD</kbd>+<kbd>Shift</kbd>+<kbd>B</kbd>) again. You can kill it by running the `Kill Build VS Code` task or pressing <kbd>Ctrl</kbd>+<kbd>D</kbd> in the task terminal.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -222,25 +226,26 @@ UI in the browser, extensions run in code server (NodeJS):
 .\scripts\code-server.bat --launch
 ```
 
-You can identify the development version of VS Code ("Code - OSS") by the following icon in the Dock or Taskbar:
-
-[![VS Code default icon](https://i.imgur.com/D2CeX0y.png)](https://i.imgur.com/D2CeX0y.png)
+You can identify the development version of the PearAI App ("Code - OSS") by the following icon in the Dock or Taskbar:
 
+[![PearAI App default icon](https://i.imgur.com/D2CeX0y.png)](https://i.imgur.com/D2CeX0y.png)
```
Comment (Fryingpannn) [inline CONTRIBUTING.md:231]:
```suggestion
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -222,25 +226,26 @@ UI in the browser, extensions run in code server (NodeJS):
 .\scripts\code-server.bat --launch
 ```
 
-You can identify the development version of VS Code ("Code - OSS") by the following icon in the Dock or Taskbar:
-
-[![VS Code default icon](https://i.imgur.com/D2CeX0y.png)](https://i.imgur.com/D2CeX0y.png)
+You can identify the development version of the PearAI App ("Code - OSS") by the following icon in the Dock or Taskbar:
```
Comment (Fryingpannn) [inline CONTRIBUTING.md:229]:
```suggestion
You can identify the development version of the PearAI App by finding the Pear logo in your Dock or Taskbar.
```

**Review by Fryingpannn — APPROVED** [review]

**Pull request merged**